pages/dkp/kommander/2.3/clusters/custom-domain-certificate/configuration/index.md

```diff
@@ -0,0 +1,141 @@
+
+    ```yaml
+    cat <<EOF | kubectl -n <workspace_namespace> --kubeconfig <management_cluster_kubeconfig> patch \ 
+    kommandercluster <cluster_name>  --type='merge' --patch-file=/dev/stdin
+    spec:
+      ingress:
+        hostname: <cluster_hostname>
+        issuerRef:
+          name: <issuer_name>
+          kind: ClusterIssuer # or Issuer depending on the issuer config
+    EOF
+    ```
+
+    Another option is to use a manually created certificate that is **customized for your hostname**. To do so, create a secret holding the certificate on the target cluster. Reference that secret in the `certificateSecretRef` field and the custom domain name in the `hostname` field of the target cluster:
+
+    ```yaml
+    cat <<EOF | kubectl -n <workspace_namespace> --kubeconfig <management_cluster_kubeconfig> patch \ 
+    kommandercluster <cluster_name>  --type='merge' --patch-file=/dev/stdin
+    spec:
+      ingress:
+        hostname: <cluster_hostname>
+        certificateSecretRef:
+          name: <secret_name>
+    EOF
+    ```
+
+<p class="message--note"><strong>NOTE: </strong>It is not possible to configure the namespace of the secret with a command. Ensure the secret is stored in the workspace namespace of the target cluster.</p>
+
+### Example: Configure a custom certificate with Let's Encrypt
+
+Let's Encrypt is one of the Certificate Authorities (CA) supported by cert-manager. To set up a Let's Encrypt certificate, provide an `Issuer` or `ClusterIssuer` on the target cluster in the `issuerRef` field.
+
+1.  Create the Let's Encrypt ACME cert-manager issuer:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: cert-manager.io/v1
+    kind: ClusterIssuer
+    metadata:
+      name: custom-acme-issuer
+    spec:
+      acme:
+        email: <your_email>
+        server: https://acme-v02.api.letsencrypt.org/directory
+        privateKeySecretRef:
+          name: kommander-acme-issuer-account
+        solvers:
+          - dns01:
+              route53:
+                region: us-east-1
+                role: arn:aws:iam::YYYYYYYYYYYY:role/dns-manager
+    EOF
+    ```
+
+1.  Configure the Management cluster to use your `custom-domain.example.com` with a certificate issued by Let's Encrypt.
+
+    ```yaml
+    cat <<EOF | kubectl -n kommander --kubeconfig <management_cluster_kubeconfig> patch \ kommandercluster host-cluster  --type='merge' --patch-file=/dev/stdin
+    spec:
+      ingress:
+        hostname: custom-domain.example.com
+        issuerRef:
+          name: custom-acme-issuer
+          kind: ClusterIssuer
+    EOF
+    ```
+
+## Verify the status of the configuration and troubleshoot in case of errors
+
+If you want to ensure the customization for a domain or a certificate is completed, or if you want to obtain more information if the customization fails, display the status information for the `KommanderCluster`.
```

Comment:
Comment by Craig: And maybe the command to run here is `kubectl describe KommanderCluster -n <workspace_name> <cluster_name> ?

Reply:
@mhrabovcin

Reply:
For consistency, I'd go with `kubectl describe kommandercluster -n <workspace_name> <cluster_name>`

Reply:
addressed in caa80f8